prac2_login_verification_UI/main.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MaterialApp(
+  home: Loginpage(),
+));
+
+class Loginpage extends StatefulWidget {
+  const Loginpage({Key? key}) : super(key: key);
+
+  @override
+  State<Loginpage> createState() => _LoginpageState();
+}
+
+class _LoginpageState extends State<Loginpage> {
+  bool value=true;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(

```

Comment:
Please try to resolve this issue. (bottom overflow)
**You may have to see how to remove overflow when keypad opens up.**

![WhatsApp Image 2022-12-23 at 10 45 42](https://user-images.githubusercontent.com/106133014/209275580-dc5075a1-4c39-4546-91bb-4738bb6d5f9e.jpeg)

Reply:
Resolved in a3c6fdf... :+1: